axelrod/tests/unit/test_tournament.py

```diff
@@ -154,20 +308,24 @@ def test_no_progress_bar_play(self):
 
 
```

Comment:
i noticed the same four tests being used about 5 times here.  how about using this?

```
    def assert_play_pbar_correct_total_and_is_finished(self, pbar, total):
        self.assertEqual(pbar.desc, 'Playing matches: ')
        self.assertEqual(pbar.total, total)
        self.assertEqual(pbar.n, total)
        self.assertTrue(pbar.disable, True)
```

if good, i would appreciate a good name suggestion.